syntax_and_semantics/case.md

```diff
@@ -71,6 +71,16 @@ elsif tmp.odd?
 end
 ```
 
+You may use `then` after the `when` condition. This is most frequently used to place the body of the `when` on a single line.
```

Comment:
I'd say "You may use then after the when condition to place the body on a single line."